src/index.cljs.hl

```diff
@@ -26,23 +26,27 @@
 (def route-from-values (cell= (str "#" (v/safe-value max-threads "") "/" (v/safe-value overhead "") "/" (v/safe-value runtime ""))))
 (def route-updates (cell= (set! (.-hash (.-location js/document)) route-from-values)))
 
-(defelem number-input [{:keys [to name]} children]
+(defelem number-input [{:keys [to name min]} children]
   (defn from-string [value] (js/parseFloat value))
   (defn validity [value] (not (js/isNaN (from-string value))))
   (defn update-to! [value] (reset! to value))
   (let [val (cell (:value @to))
         update-to! (partial reset! to)
         update-val! (partial reset! val)
         valid (cell= (validity val))
+        correct? (fn [v min] (and (not (empty? v))
+                                  (>= (from-string v) min)))

```

Comment:
The reasoning behind this is that I would like to propagate error states out of the input element so that other things can decide if they care or not.